web/html/src/components/action-schedule.js

```diff
@@ -198,6 +197,13 @@ class ActionSchedule extends React.Component<ActionScheduleProps, ActionSchedule
     </div>);
   }
 
+  renderEmptyMaintWindowsWarning = () => {
```

Comment:
Just some really minor nitpick. This function uses `alert-info`, but is called `...Warning`.
Maybe call it `...Info`?.
This is just a suggestion, I am totally fine if you ignore it :smile:

Reply:
Why not, `info` sounds even better.